pages/doc/proxies_preprocessor_rules.md

```diff
@@ -1069,6 +1069,67 @@ Extract a string from a span name, source name, or a span tag value and create a
   # optional, omit if you plan on just extracting the tag leaving the metric name intact
 ```
 
+### spanRenameTag
+Renames a span tag. The renaming does not affect the values stored in a span.
+
+<font size="3"><strong>Parameters</strong></font>
+
+<table>
+<colgroup>
+<col width="15%" />
+<col width="85%" />
+</colgroup>
+<thead>
+<tr>
+<th>Parameter</th>
+<th>Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td>action</td>
+<td>spanRenameTag</td>
+</tr>
+<tr>
+<td>key</td>
+<td>The span tag to be renamed.</td>
+</tr>
+<tr>
+<td>newkey</td>
+<td>The new name for the span tag.</td>
+</tr>
+<tr>
+<td>match (optional)</td>
+<td>If specified, renames a span tag if its value matches this regular expression.</td>
+</tr>
+<tr>
+<td>firstMatchOnly (optional)</td>
+<td>If set to true, renames only the first matching span tag. Default is false.</td>
+</tr>
+</tbody>
+</table>
+
+<font size="3"><strong>Example</strong></font>
+
+```yaml
+## rename the "guid:x-request-id" span tag to "guid-x-request-id" by 
+## removing the invalid punctuations (":") to prevent it from being blacklisted.
```

Comment:
...removing the invalid punctuation (not punctuations)

Reply:
+1